LICENSE

```diff
@@ -0,0 +1,19 @@
+#' Copyright <2019> <Qualitative Data Repository, Syracuse University>
```

Comment:
Any reason you're using the #' `roxygen` comments? Since this is in a separate file, the license can just be plain text, no?

Reply:
No reason to use comments I don't think.

Reply:
Will it mess with your CRAN submission if I take these out? Otherwise this is good to go.

Reply:
No I don't think so. Once it's accepted (or not) I can repair it accordingly.